src/main/java/org/folio/inventory/services/InstanceRelationshipsService.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.inventory.services;
+
+import java.util.List;
+
+import org.folio.inventory.storage.external.CollectionResourceClient;
+import org.folio.inventory.storage.external.CqlQuery;
+import org.folio.inventory.storage.external.MultipleRecordsFetchClient;
+
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+
+public class InstanceRelationshipsService {
```

Comment:
How about removing duplication? Work with only one fetchClient and pass relationshipsFetchClient'/precedingSucceedingFetchClient as a parameter to the  constructor?

Reply:
All these fetch client requires different CQL generation methods. So it is not possible to re-use code. Also, the intent of this class to hide the fetching logic.

Is it fine?